desktop/ui/src/main/java/org/datacleaner/windows/AnalysisJobBuilderWindowImpl.java

```diff
@@ -505,10 +512,10 @@ public void updateStatusLabel() {
                     setStatusLabelWarning();
                 }
             } catch (Exception ex) {
+                executeable = false;
```

Comment:
I tried it in 4.0.11 and in 4.5 and without any configured column in ValueDistributionAnalyzer the job can still be executed. However, there is no error/exception mentioned (just nothing happens / no rows processed).